util/src/main/java/org/killbill/billing/util/security/shiro/dao/SessionModelDao.java

```diff
@@ -32,7 +32,7 @@
 
     private final Serializer<Map> serializer = new DefaultSerializer<Map>();
 
-    private Long recordId;
```

Comment:
We should keep reading it (you'll need to update the query too) since it's in the database.